index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page Capstone Project</title>
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css">
+ 
+</head>
+<body>
+    <header id="dsktop-header">
+        <nav id="nav1" title="first-navigation">
+          <ul>
+            <li><a class="fa-brands fa-facebook-f"></a></li>
+            <li><a class="fa-brands fa-twitter"></a></li>
+            <li><a href="#" class="nav1item">Swahili</a></li>
+            <li><a href="#" class="nav1item">My Page</a></li>
+            <li><a href="#" class="nav1item">Logout</a></li>
+          </ul>
+        </nav>
+         
+        <nav id="nav2" title="second-navigation"> 
+          <a class="main-logo" href="#home"><img src="./sources/logo.png" alt="logo"></a>
+          
+          <a class="nav2Item" href="./about.html">About</a>
+          <a class="nav2Item" href="#packages">Our Packages</a>
+          <a class="nav2Item" href="./about.html">Join</a>
+          <a class="nav2Item" href="#sponsors">Sponsor</a>
+          <a class="nav2Item" href="./about.html">News</a>
+          <a class="mainItem" href="./about.html">Contributions</a> 
+        </nav>   
+    </header>
+
+    <header id="mobile-header">
+      <nav class="mobile-nav"> 
+          <ul onclick="menuPopup()" id="navList" title="navigation-list">
+            <li><a class="navItem" href="./about.html">About</a></li>
+            <li><a class="navItem" href="#packages">Our Packages</a></li>
+            <li><a class="navItem" href="./about.html">Join</a></li>
+            <li><a class="navItem" href="#sponsors">Sponsor</a></li>
+            <li><a class="navItem" href="./about.html">News</a></li>
+            <li><a class="navItem navItem-main" href="./about.html">Contributions</a></li>
+          </ul> 
+      </nav>   
+      <nav>
+        <div onclick="menuPopup()" id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div> 
+        <div onclick="menuPopup()" id="closeButton"> &times; </div>
+      </nav>
+    </header>
```

Comment:
[OPTIONAL] The previous reviewer didn't notice so I will ask it as optional. You need to implement a mobile nav like on the design. It needs to be transparent. Your has white background.
<img width="436" alt="Screenshot 2023-03-17 at 9 32 10 PM" src="https://user-images.githubusercontent.com/25887253/225989843-e79a443a-3c60-4b87-b11f-f62aef33ec41.png">
<img width="501" alt="Screenshot 2023-03-17 at 9 32 41 PM" src="https://user-images.githubusercontent.com/25887253/225989849-c960fd71-6020-48e7-ae09-ff7a5fcd537d.png">